go.mod

```
@@ -1,123 +1,129 @@
 module github.com/devfile/devworkspace-operator
 
-go 1.23.0
+go 1.24.0
+
+toolchain go1.24.1
```

Could we change it to:

```suggestion
toolchain go1.24.4
```